### PR TITLE
Spoi 12801 metrics training app

### PR DESCRIPTION
**Pull request by yogidevendra**

### CODE REVIEW

**Comment by yogidevendra** [comment]
@patilvikram Kindly review this.

**Comment by yogidevendra** [comment]
@chaithu14 could you please review this?

File: training/random-to-hdfs/pom.xml [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.datatorrent.apps</groupId>
+  <version>0.9</version>
+  <artifactId>random-to-hdfs-app</artifactId>
+  <packaging>jar</packaging>
+
+  <properties>
+    <!-- change this if you desire to use a different version of Apex Core -->
+    <apex.version>3.7.0-SNAPSHOT</apex.version>
```
Comment (chaithu14) [inline training/random-to-hdfs/pom.xml]: Same here?

File: training/random-to-hdfs/pom.xml [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+      <groupId>org.apache.apex</groupId>
+      <artifactId>apex-engine</artifactId>
+      <version>${apex.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-common</artifactId>
+      <version>2.6.0</version>
+      <scope>provided</scope>
```
Comment (chaithu14) [inline training/random-to-hdfs/pom.xml]: Same comment.

**Comment by chaithu14** [comment]
LGTM

**Pull request merged**